### PR TITLE
fix doc

### PR DESCRIPTION
**Pull request by FardaleM**
Fix errors in the documentation.

### CODE REVIEW

File: src/data/CCGraph.mli [hunk 1]
```diff
@@ -35,17 +34,12 @@ type 'a iter_once = 'a iter
...
-  val map : ('a -> 'b) -> 'a t -> 'b t
-  val filter_map : ('a -> 'b option) -> 'a t -> 'b t
-  val iter : ('a -> unit) -> 'a t -> unit
-  val fold : ('b -> 'a -> 'b) -> 'b -> 'a t -> 'b
-  val to_list : 'a t -> 'a list
-end
+  val return : 'a -> 'a t val ( >>= ) : 'a t -> ('a -> 'b t) -> 'b t val map :
+  ('a -> 'b) -> 'a t -> 'b t val filter_map : ('a -> 'b option) -> 'a t -> 'b t
+  val iter : ('a -> unit) -> 'a t -> unit val fold : ('b -> 'a -> 'b) -> 'b ->
+  'a t -> 'b val to_list : 'a t -> 'a list end
```
Comment (c-cube) [inline src/data/CCGraph.mli:42]: what happened here? :)

Comment (FardaleM) [reply src/data/CCGraph.mli:42]: I think it is ocamlformat. I will correct this and force push.

File: src/data/CCGraph.mli [hunk 2]
```diff
@@ -59,29 +53,28 @@ type ('v, 'e) graph = ('v, 'e) t
 val make : ('v -> ('e * 'v) iter) -> ('v, 'e) t
 (** Make a graph by providing the children function. *)
 
-type 'v tag_set = {
-  get_tag: 'v -> bool;
-  set_tag: 'v -> unit;  (** Set tag for the given element *)
-}
+type 'v tag_set = { get_tag: 'v -> bool; set_tag: 'v -> unit;  (** Set tag for
+                                                                   the given
+                                                                   element *) }
```
[the remark this answers is not in the document]
Comment (FardaleM) [reply src/data/CCGraph.mli:58]: Yes, I think so.

**Comment by FardaleM** [comment]
The mistake with ocamlformat should be fixed now.

**Comment by c-cube** [comment]
Thanks :). Always the keen eye…

**Pull request merged**